Gemfile

```diff
@@ -30,6 +30,7 @@ gem 'slim-rails'                                                   # slim templa
 gem "sprockets", "~> 3.7.2"                                        # sprockets is a rack-based asset packaging system that concatenates and serves javascript, scss, etc
 gem 'sucker_punch', '~> 2.0'                                       # asynchronous processing library
 gem 'uglifier', '>= 1.3.0'                                         # compressor for javascript assets
+gem 'ffi', '>= 1.9.24'
```

Comment:
`ffi` is only required by the `listen` gem which is only used in development.  Let's...

- move this line to the development group
- tweak the gem version to use ~>
- add a comment like the other gems something like "an foreign function interface implementation for Ruby"

Reply:
Done :slightly_smiling_face: